### PR TITLE
Translate the document to English

### PR DESCRIPTION
**Pull request by NoNormalCreeper**
I don't know if I should add my name in the *license* section, so I'm not adding it yet.

### CODE REVIEW

File: README_EN.md [hunk 1]
```diff
@@ -2,6 +2,219 @@
 
 A short tutorial about updating `CentOS 8` / `CentOS Stream 8` to `CentOS Stream 9`
 
-[中文](README.md) | English(WIP)
+[中文](README.md) | [English](README_EN.md)
 
 ---
+
+Listen, dude! Did you know? In December 31st, 2021, `CentOS 8` ~~was finished! 🎉🎉🎉~~
```
Comment (Ink-33) [inline README_EN.md]:
``` markdown
It was December 31st, 2021 that `CentOS 8` reached end of life!  🎉🎉🎉  
```
Please note the two spaces at the end of the sentence.

File: README_EN.md [hunk 2]
```diff
@@ -2,6 +2,219 @@
...
 ---
+
+Listen, dude! Did you know? In December 31st, 2021, `CentOS 8` ~~was finished! 🎉🎉🎉~~
+Are you still using `CentOS 8`? Why not update to `CentOS Stream 8`, `Rocky Linux 8` or `AlmaLinux` ?
+
+Wait, don't you hear that `CentOS Stream 9` has already released? There are even `Nodejs16` , `Golang1.17.5`, `kernel5.14` and `glic 2.34` package indexes available!
+
+If you pursue new features and a `dnf` package manager, `CentOS Stream 9` is probably your second best choice! (The first one is still `Fedora`)
+
+Unfortunately my VPS provider doesn't provide `CentOS Stream 9` image, or even `CentOS Stream 8` image, so I have to use the outdated `CentOS 8` image, isn't that bad?
```
Comment (Ink-33) [inline README_EN.md]: But my VPS provider doesn't provide a `CentOS Stream 9` image, or even a `CentOS Stream 8` image, so I have to use an outdated `CentOS 8` image, wouldn't that be bad?

File: README_EN.md [hunk 3]
```diff
@@ -2,6 +2,219 @@
...
+Listen, dude! Did you know? In December 31st, 2021, `CentOS 8` ~~was finished! 🎉🎉🎉~~
+Are you still using `CentOS 8`? Why not update to `CentOS Stream 8`, `Rocky Linux 8` or `AlmaLinux` ?
+
+Wait, don't you hear that `CentOS Stream 9` has already released? There are even `Nodejs16` , `Golang1.17.5`, `kernel5.14` and `glic 2.34` package indexes available!
+
+If you pursue new features and a `dnf` package manager, `CentOS Stream 9` is probably your second best choice! (The first one is still `Fedora`)
+
+Unfortunately my VPS provider doesn't provide `CentOS Stream 9` image, or even `CentOS Stream 8` image, so I have to use the outdated `CentOS 8` image, isn't that bad?  
+
+Don't worry, with the method provided in this article, you can also upgrade to `CentOS Stream 9` on `CentOS 8`, with all your data kept and no need to reinstall system!
```
Comment (Ink-33) [inline README_EN.md]: Don't worry, with the method provided in this article, you can also upgrade to `CentOS Stream 9` from `CentOS 8`, with all your data kept and not have to reinstall your system!

File: README_EN.md [hunk 4]
```diff
@@ -2,6 +2,219 @@
...
+
+## How to use
+
+**WARNING**: Upgrading a large version may cause the software or even the system to not work properly, so please be careful!
```
Comment (Ink-33) [inline README_EN.md]: Upgrading a major version may cause the software or even the system to fail to operate properly, so please be careful!

File: README_EN.md [hunk 5]
```diff
@@ -2,6 +2,219 @@
...
+
+Unfortunately my VPS provider doesn't provide `CentOS Stream 9` image, or even `CentOS Stream 8` image, so I have to use the outdated `CentOS 8` image, isn't that bad?  
+
+Don't worry, with the method provided in this article, you can also upgrade to `CentOS Stream 9` on `CentOS 8`, with all your data kept and no need to reinstall system!
+
+## How to use
+
+**WARNING**: Upgrading a large version may cause the software or even the system to not work properly, so please be careful!  
+
+**TIPS**: Please read the whole article before you start the operation. If you encounter a problem, you can open an `issue` and create a `pull request` for a better solution.
```
Comment (Ink-33) [inline README_EN.md]: Fell free to open an `issue` when you encounter a problem. If you would like to share a better solution, please submit a `pull request` to help us improve the documention.

File: README.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 
 CentOS 8 / CentOS Stream 8 升级 CentOS Stream 9 简明教程
 
-[中文](README.md) | English(WIP)
+[中文](README.md) | [English](README_EN.md)
 
 ---
```
Comment (Ink-33) [inline README.md:8]: Please help me change L9 to 
``` markdown 
哦，我的老伙计！你知道吗？`CentOS 8`在2021年12月31日完蛋啦！🎉🎉🎉  
```
Please note the two spaces at the end of the sentence.

**Review by Ink-33 — CHANGES_REQUESTED** [review]

**Comment by NoNormalCreeper** [comment]
Modified.

**Review by Ink-33 — APPROVED** [review]

**Pull request merged**